src/stream/src/executor/hash_agg.rs

```diff
@@ -116,6 +118,12 @@ struct HashAggExecutorExtra<K: HashKey, S: StateStore> {
 
     /// The maximum size of the chunk produced by executor at a time.
     chunk_size: usize,
+
+    /// Map group key column idx to its position in group keys.
+    group_key_invert_idx: Vec<isize>,
```

Comment:
can we move the `ColIndexMapping` to common which is widely used in frontend? :thinking: we can do it later,